meta/conf/distro/include/ptest-packagelists.inc

```diff
@@ -112,6 +112,7 @@ PTESTS_PROBLEMS:append:riscv64 = "valgrind-ptest"
 #    numactl-ptest \ # qemu not (yet) configured for numa; all tests are skipped
 #    libseccomp-ptest \ #  tests failed: 38; add to slow tests once addressed
 #    python3-numpy-ptest \ # requires even more RAM and (possibly) disk space; multiple failures
+#    run-postinsts-ptest \ # depends upon bash-ptest which is non-deterministic by design
```

Comment:
What does "non-deterministic" mean in this context? I'm guess that's probably from an opinion from an ML or commit discussion; right?

Reply:
In this case it means the test is not guaranteed to complete in a specific amount of time like the other ptests, in this case it is apparently intentional.

Tests in the PTESTS_FAST variable complete in <30s, PTESTS_SLOW in >30s, and PTESTS_PROBLEMS could fail, be inconsistent, or in bash's case, complete in an inconsistent amount of time.

I know that run-postinsts is dependent on the bash test, which already had the comment that it was "non-deterministic by design," so I made the assumption that ours would also have to be non-deterministic as a consequence

Reply:
Is `run-postinsts-ptest` depending on `bash-ptest` through [this dependency](https://github.com/ni/meta-nilrt/blob/nilrt/master/kirkstone/recipes-devtools/run-postinsts/run-postinsts_%25.bbappend#L31) we added in the meta-nilrt .bbappend? I wouldn't have expect that to cause the -ptests to depend on each other.

Conceptually, I can't think of a reason that any `-ptest` subpackage would depend upon another.

Reply:
Yeah. That would make sense. I misunderstood what was happening with the RDEPENDS there. It doesn't require bash-ptest, it requires bash because presumably it runs shell scripts.

With that said
1) I'm not sure if it should stay in PTESTS_PROBLEMS or go in another one
2) on further reflection, this is probably something that should be in upstream

Reply:
Well, the reason upstream won't hit this warning is because it's meta-nilrt that [adds](https://github.com/ni/meta-nilrt/blob/nilrt/master/hardknott/recipes-devtools/run-postinsts/run-postinsts_%25.bbappend#L11) the ptests to `run-postinst`. 

Looking at the ptests themselves, I don't see them taking any real amount of time. So [`PTEST_FAST`](https://github.com/ni/openembedded-core/blob/8663257cc64fd138addf9c22ff9e5516316a5031/meta/conf/distro/include/ptest-packagelists.inc#L6) is probably the best place for it.

Since we're adding this logic in our layer, we should append to PTEST_FAST in our layer as well. Since the ptest-packagelists include gets pulled into the recipe targets [via the ptest.bbclass](https://github.com/ni/openembedded-core/blob/nilrt/master/kirkstone/meta/classes/ptest.bbclass#L23), I think you can just add `PTEST_FAST += ${PN}` to the run-postsinsts.bbappend in meta-nilrt.

Reply:
Alright, I'll make that change